### ISSUE TITLE
does reactive-table has Row Expander functionality ?

### ISSUE THREAD

**Issue by sharathbangera6** [aslagle/reactive-table#350]
Hi,

Wanted to know if reactive-table provides row-expander functionality ? any idea on how to implement the same if it doesn't provide ?

**Comment by aslagle** [comment]
No, sorry. And unfortunately I don't think you could implement it without modifying the package.

**Comment by datanotion** [comment]
@sharathbangera6 you could consider doing it this way:  https://github.com/aslagle/reactive-table/pull/296 until there is an officially supported option.  I know that the gentleman that wrote it is planning on eventually writing it as a tmpl as @aslagle suggested, but it's an option for now (I'm using it).   You'd just have to be careful when updating the core reactive-table package.

**Comment by ClarenceL** [comment]
I've added a new pull request

You can try out the code here: https://github.com/ClarenceL/reactive-table/tree/feature-child-tmpl

It's pretty simple, on the column you want the expand button, for the fields item, add the setting `expandButton: true`

Then add to the root settings object  `
children: {
        tmpl: Template."your template name"
}`

See the added readme CHILD_TABLE_README.md for more details
